x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/elasticsearch/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,227 @@
+                    id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatLinkText"
+                    defaultMessage="the instructions here"
+                  />
+                </EuiLink>
+              )
+            }}
+          />
+        </p>
+      </EuiText>
+    )
+  };
+
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the Elasticsearch module in Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+            metricbeat modules enable elasticsearch-xpack
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.enableMetricbeatModuleDescription"
+              defaultMessage="By default the module will collect Elasticsearch monitoring metrics from {url}.
+              If the local Elasticsearch server has a different address,
+              you must specify it via the hosts setting in the {module} file."
+              values={{
+                module: (
+                  <Monospace>modules.d/elasticsearch-xpack.yml</Monospace>
+                ),
+                url: (
+                  <Monospace>http://localhost:9200</Monospace>
+                )
+              }}
+            />
+          </p>
+        </EuiText>
+      </Fragment>
+    )
+  };
+
+  const configureMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.configureMetricbeatTitle', {
+      defaultMessage: 'Configure Metricbeat to send to the monitoring cluster'
+    }),
+    children: (
+      <Fragment>
+        <EuiText>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.configureMetricbeatDescription"
+            defaultMessage="Make these changes in your {file}."
+            values={{
+              file: (
+                <Monospace>metricbeat.yml</Monospace>
+              )
+            }}
+          />
+        </EuiText>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`
+output.elasticsearch:
+  hosts: ["${esMonitoringUrl}"] ## Monitoring cluster
+
+  # Optional protocol and basic auth credentials.
+  #protocol: "https"
+  #username: "elastic"
+  #password: "changeme"
+`}
+        </EuiCodeBlock>
+      </Fragment>
+
+    )
+  };
+
+  const startMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.startMetricbeatTitle', {
+      defaultMessage: 'Start Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`
+metricbeat -e
```

Comment:
When I do this step on my Mac, I have to also do the chown steps otherwise I get errors (see https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-starting.html)